src/main/java/org/mule/extension/introspection/declaration/fluent/BaseDeclaration.java

```diff
@@ -70,6 +73,59 @@ public T addCapability(Object capability)
     }
 
     /**
+     * Returns a map with the currently set model properties. Notice
+     * that this map is mutable and not thread-safe. This method
```

Comment:
Not sure if the "This method... " section adds value, but if you want to keep it, add a paragraph separator before it.